sass/components/_multi-select.scss

```diff
@@ -1,12 +1,12 @@
 @charset 'UTF-8';
 
 .multi-select__container {
-    display: table;
+    display: flex;
+    justify-content: space-between;
     width: 100%;
     padding: 0 15px;
 
     .btn {
-        display: table-cell;
         min-width: 60px;
```

Comment:
we can also remove `min-width` and `vertical-align` since they do not have any effect (other than restricting the width of the button)